src/query/service/src/interpreters/access/accessor.rs

```diff
@@ -48,13 +48,6 @@ impl Accessor {
         Accessor { accessors }
     }
 
-    pub async fn check(&self, plan: &PlanNode) -> Result<()> {
```

Comment:
Also need to remove the `async fn check` from the Accessor trait.

Reply:
Addressed in https://github.com/datafuselabs/databend/pull/7762/commits/264a6491a1d2c57b69a4d808619dbd0d6f0d9dad